dart/dynamics/detail/JointAspect.hpp

```diff
@@ -117,14 +125,23 @@ struct JointProperties
   /// Actuator type
   ActuatorType mActuatorType;
 
+  /// Mimic joint
+  Joint* mMimicJoint;
+
+  /// Mimic joint properties
+  double mMultiplier, mOffset;
+
   /// Constructor
   JointProperties(const std::string& _name = "Joint",
              const Eigen::Isometry3d& _T_ParentBodyToJoint =
                                  Eigen::Isometry3d::Identity(),
              const Eigen::Isometry3d& _T_ChildBodyToJoint =
                                  Eigen::Isometry3d::Identity(),
              bool _isPositionLimitEnforced = false,
-             ActuatorType _actuatorType = DefaultActuatorType);
+             ActuatorType _actuatorType = DefaultActuatorType,
+             Joint* _mimicJoint = nullptr,
+             double _multiplier = 1.0,
+             double _offset = 0.0);
```

Comment:
```suggestion
             double _mimicMultiplier = 1.0,
             double _mimicOffset = 0.0);
```

to make it clear that these are properties specifically related to the mimic joint actuator type.